File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int t=nums.size();
        int i=0,sum=0;
        for(i=0;i<t;i++){
            sum+=nums[i];
        }
        int total_sum= t*(t+1)/2;
        int missing=total_sum-sum;
     return missing;
    }

};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums.sort()
        for i in range(len(nums)):
            if i!=nums[i]:
                return i
        return len(nums)
        # if len(nums)==(1+nums[len(nums)-1]):
        #     return len(nums)
        # else:
        #     t=len(nums)
        #     x=t*(t+1)//2
        #     z=x-sum(nums)
        #     return z
